### ISSUE TITLE
weight decay and lfw accuracy

### ISSUE THREAD

**Issue by 406747925** [TMaysGGS/MobileFaceNet-Keras#2]
请问模型权重需要l2正则化吗，训练得到lfw精度可以达到多少？

**Comment by TMaysGGS** [comment]
因为我这边硬件受限一次完整的训练epoch需要半个多月，暂时没必要改weight decay都是用的默认值。个人理解，后续如果训练出一个较好版本且模型产生过拟合现象了再加也不迟。
